src/inference/src/ie_core.cpp

```diff
@@ -245,32 +245,95 @@ class CoreImpl : public ie::ICore, public std::enable_shared_from_this<ie::ICore
             std::shared_ptr<ie::ICacheManager> _cacheManager;
         };
 
-        bool flag_allow_auto_batching = true;
+        // Core default global properties, which will not set to any plugins.
+        const ov::AnyMap default_global_properties = {ov::force_tbb_terminate(false)};
+
+        // Core default plugins properties, which will set to specified or all plugins.
+        const ov::AnyMap default_plugins_properties = {ov::cache_dir(""),
+                                                       ov::hint::allow_auto_batching(true),
+                                                       ov::auto_batch_timeout(1000)};
```

Comment:
this is already hardcoded in auto batch:
```c++
    _config[CONFIG_KEY(AUTO_BATCH_TIMEOUT)] = "1000";  // default value, in ms
```
Can we avoid duplication? Maybe propagate this value to auto-batch
I suppose in `update_config` you already set this value and it later comes to BATCH plugin, is not it? If yes, we can remove default value from BATCH plugin

Reply:
Not exact, `update_config()` only updates the core-configs that have ever been set by core.set_property() without device name into plugin, other core-config will not set to plugin.
So if user didn't call core.set_property(ov::auto_batch_timeout(x)), it still depends on auto plugin has own default value.
BTW, the default value in `default_plugins_properties` can be used for core.get_property().

Reply:
For default value(such as AUTO_BATCH_TIMEOUT) in plugins, there will other PRs to resolve them.